benchmark/Main.hs

```diff
@@ -89,7 +89,7 @@ bgroupFold :: Benchmark
 bgroupFold = do
     let testList   = [1..100000] :: [Int]
     let flipFoldl' = flipfoldl' HashSet.insert mempty
-    let ghcFoldl'  = foldl' (\hashSet element -> HashSet.insert element hashSet) mempty
+    let ghcFoldl'  = foldl' (flip HashSet.insert) mempty
```

Comment:
Let's not use `flip` here and return back to the previous version. `flipfoldl' f` is `foldl' (flp f)` so in this benchmarks we specifically measured the performance overhead of additional `flip` call.